parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -482,71 +482,6 @@ isMarinerOrAzureLinux() {
     return 1
 }
 
-installJq() {
```

Comment:
isnt this used anywhere else ?

Reply:
Note for future: installJq function used to be here but was moved to `cse_helpers.sh`, evalPackageDownloadURL moved up after removal